ndk-build/src/apk.rs

```diff
@@ -155,6 +157,20 @@ impl<'a> UnalignedApk<'a> {
         Ok(())
     }
 
+    pub fn add_extra_libs(&self, target: Target) -> Result<(), NdkError> {
+        let abi_dir = Path::new("lib").join(target.android_abi());
```

Comment:
If going back to this PR we should make `"lib"` configurable in case an Android Rust APK resides inside a larger project where the `lib/` folder is already used for other purposes.

It should probably also be relative to the directory containing `Cargo.toml`, this is now relative to the directory where `cargo apk` is invoked (ie. `android-ndk-rs/` when calling `cargo apk build -p ndk-examples --example hello_world` or `android-ndk-rs/ndk-examples` when calling `cd ndk-examples &&  `cargo apk build --example hello_world`).

Reply:
Ok, noted, I'll take that into account.

Reply:
Do you know how I can get the path to the directory containing `Cargo.toml`?

Reply:
As with the `res` and `assets` directories, this path is retrieved using:

```rust
                    &self
                        .cmd
                        .manifest()
                        .parent()
                        .expect("invalid manifest path")
```

I'd put that in a new `let crate_path =` variable, use it in both places, and pass it to `add_extra_libs` together with a similar `self.manifest.extra_libs...` optional path.

Reply:
Ok, so I reused that part without noticing, I'm going to refactor that to use a `crate_path` variable now.

Reply:
I think I've done all the requested changes now.